metadata/test/test.jsonl

```diff
@@ -0,0 +1,2 @@
player in the high school class of 2007 by Rivals.com and one of the top prospects for the 2008 NBA draft.\nThe young team concluded the season 21–12 (10–6) and reached the second round of the NCAA tournament.\nPlayers.\nRoster.\n1\n2\n3\n4\nIncoming signees.\n2007 saw the Wildcats put together one of the nation's strongest signing classes, ranked 1st according to Scout.com, 2nd according to Rivals.com, and 5th according to ESPN.com. PF Beasley (Washington, D.C.) is the standout of the class, but is far from being its only talented member. The Wildcats signed two talented PGs in Fred Brown (West Palm Beach, Florida) and Jacob Pullen (Maywood, Illinois) and added SF Andre Gilbert (Brooklyn Park, Minnesota), a JUCO transfer with Division I experience. KSU signed a pair of talented players from the Patterson School in North Carolina in SF Dominique Sutton (Durham, North Carolina) and PF Jamar Samuels (Washington, D.C.). Both will attend prep school for the Fall 2007 semester to improve their grades and will join the team at the start of the Spring 2008 semester.\nThough scheduled to graduate from high school in the Spring of 2007, SF Bill Walker (Huntington, West Virginia) finished high school early and enrolled at KSU for the 2007 Spring Semester and was allowed to join the team immediately in December of the 2006–07 season. However, he was injured in just his sixth game at KSU and opted to redshirt for the remainder of the season. He will resume his freshman season of eligibility in 2007–08. PF Ron Anderson (Upper Marlboro, Maryland), a long time AAU teammate of Beasley's, rounded out the class when he was offered a scholarship after a strong AAU showing in the Summer of 2007.\nSchedule.\n!colspan=9| Exhibition\n!colspan=9| Regular season\n!colspan=9| 2007 Old Spice Classic\n!colspan=9| Big 12\/Pac-10 Hardwood Series\n!colspan=9| 13th Annual Jimmy V Classic\n!colspan=9| K-State Holiday Classic presented by Subway\n!colspan=9| Big 12 Regular season\n!colspan=9| Phillips 66 2008 Big 12 Championship tournament\n!colspan=9|2008 NCAA tournament\nPost-season.\nBig 12 tournament.\nThe Wildcats went on to a 10–6 record in conference play, earning a number 3 seed in the 2008 Big 12 men's basketball tournament at the Sprint Center in Kansas City, Missouri. The Wildcats faced the No. 6 seed Texas A&amp;M Aggies and lost 60–63.\nNCAA tournament.\nFirst round.\nThe Wildcats earned a berth in the 2008 NCAA tournament as the No. 11 seed in the Midwest Region. In the first round they beat the No. 6 seed USC Trojans, 80–67. The victory over the Trojans was the Wildcats' first win in a tournament game since beating No. 1 seed Purdue, 73–70, in the Midwest Regional semifinals on March 25, 1988.\nSecond round.\nThe Wildcats faced the Wisconsin Badgers from the Big Ten Conference in the second round and were soundly defeated, 55–72.\nMichael Beasley scored 23 points and grabbed 13 rebounds for his 28th double-double of the season, but he had only six after halftime."}
```

Comment:
I think the test file needs to escape backslashes and `\n` since when I ran the script it failed due to corrupt records...when replacing the text with generic hello worlds it worked though so I think it'll be fine after escaping!

Reply:
*

Reply:
escaped all backslashes and special chars, added `metadata/test/escape_special_chars.py` to handle this